Makefile

```diff
@@ -757,16 +757,17 @@ podman-v%.msi: test/version/version
 		-o $@ contrib/msi/podman.wxs $(DOCFILE)/pages.wsx --arch x64
 
 # Checks out and builds win-sshproxy helper. See comment on GV_GITURL declaration
-.PHONY: win-sshproxy
-win-sshproxy: test/version/version
+.PHONY: gvproxy
+gvproxy: test/version/version
```

Comment:
Since this is Windows specific, to avoid confusion, we probably should rename the Makefile target to win-gvproxy